packages/ui/src/config/mantine.ts

```diff
@@ -1,6 +1,33 @@
-import { MantineTheme } from "@mantine/core";
+import { MantineThemeOverride } from "@mantine/core";
 
-export const mantineConfig: Partial<MantineTheme> = {
-  colorScheme: "dark"
+import { Tuple, DefaultMantineColor } from "@mantine/core";
+
+type ExtendedCustomColors = "primary" | DefaultMantineColor;
+
+declare module "@mantine/core" {
+  export interface MantineThemeColorsOverride {
+    colors: Record<ExtendedCustomColors, Tuple<string, 10>>;
+  }
+}
+
+export const mantineConfig: Partial<MantineThemeOverride> = {
+  fontFamily: "Lato",
+  colorScheme: "light",
+  colors: {
+    primary: [
+      "#CEFFEF",
+      "#71B7A0",
+      "#55B898",
+      "#3CB990",
+      "#28B788",
+      "#14B681",
+      "#00B87C",
+      "#11956A",
+      "#1B7A5B",
+      "#21654F"
+    ]
```

Comment:
where are these colors coming from since #31 has a different set of colors

Reply:
The shades here that are really important (which I've used so far) are `primary[0]`, the lightest/softest shade used for `<Button variant="subtle"/>` on hover, and `primary[6]` which is the actual primary color from the design system. The colors in #31 are shades of blue. Are the designers changing the color palette, because these colors here are from the old design?

Reply:
i don't think the designers are planning to change the color pallette

I'll leave this for now since it can easily be updated if need be